interfaces/acados_matlab_octave/acados_template_mex/+acados_template_mex/set_up_t_renderer.m

```diff
@@ -0,0 +1,44 @@
+function set_up_t_renderer(t_renderer_location,force)
+arguments
+    t_renderer_location
+    force=true
+end
+    message = ['\nDear acados user, we could not find t_renderer binaries,',...
+        '\n which are needed to export templated C code from ',...
+        'Matlab.\n Press any key to proceed setting up the t_renderer automatically.',...
+        '\n Press "n" or "N" to exit, if you wish to set up t_renderer yourself.\n',...
+        '\n https://github.com/acados/tera_renderer/releases'];
+    
+    if(~force)
+        In = input(message,'s');
+        if strcmpi( In, 'n')
+            error('Please set up t_renderer yourself and try again');
+        end
+    end
```

Comment:
Indent is weird here.

Reply:
Fixed

Reply:
How is it wierd? 

There is a nested if statement indentation follows the blocks.

Reply:
Right, sorry, I don't know why I was confused..